persistent/tests/test_picklecache.py

```diff
@@ -524,6 +557,22 @@ def test_minimize(self):
         for oid in oids:
             self.assertTrue(cache.get(oid) is None)
 
+    def test_minimize_turns_into_ghosts(self):
+        import gc
+        from persistent.interfaces import UPTODATE
+        from persistent._compat import _b
+        cache = self._makeOne()
+        oid = _b('oid_%04d' % 1)
+        obj = cache[oid] = self._makePersist(oid=oid, state=UPTODATE)
+        self.assertEqual(cache.cache_non_ghost_count, 1)
+
+        cache.minimize()
+        gc.collect() # banish the ghosts who are no longer in the ring
+
+        self.assertEqual(cache.cache_non_ghost_count, 0)
+
+        self.assertEqual(obj._p_state, -1)
```

Comment:
I don't recall now:  I think we'd be better off using the named constants everywhere for readability.